crates/renderer/src/tree_renderer.rs

```diff
@@ -216,11 +216,11 @@ impl TreeRenderer {
         gpu.queue.submit(Some(encoder.finish()));
 
         for (subbuffer, primitives) in primitive_writes {
-            tracing::debug!(
-                "Writing {} primitives to subbuffer {}",
-                primitives.len(),
-                subbuffer
-            );
+            // tracing::debug!(
+            //     "write {} primitives to subbuffer {}",
+            //     primitives.len(),
+            //     subbuffer
+            // );
```

Comment:
same as above

Reply:
Right, these two.

I left them because I know Nuno may be using them for debugging now, so didn't want to entirely remove them in case they are needed soon.

I suppose this goes into the general problem of should logs be removed entirely or kept around if they become needed again. In such cases I usually downgrade them to a quite low log level